src/internal/components/chart-plot/index.tsx

```diff
@@ -39,7 +39,7 @@ export interface ChartPlotProps {
   activeElementKey?: null | string | number | boolean;
   activeElementRef?: React.RefObject<SVGGElement>;
   activeElementFocusOffset?: Offset;
-  ariaLiveRegion?: React.ReactNode;
+  ariaLiveRegion?: RefObject<HTMLElement> | string;
```

Comment:
Can this be simplified to just `RefObject<HTMLElement>`?

Reply:
Or, maybe, instead of providing the `ariaLiveRegion` from the pie-chart render `<LiveRegion source={[ariaLiveRegion]} />` from the pie-chart.

Reply:
The problem is that this is used by other chart types as well, which use string. Before, this was not needed only because `ReactNode` includes string, but in fact pie/donut charts pass down the element whereas cartesian charts build the comma-separated string on their side. This even leads to subtle differences because pie/donut chart live region content does not have commas, and IMO TBH these two approaches should be unified, but here I'm trying to minimize the amount of changes.

Reply:
Yes, that's why my second comment. I just think the string | ref API is a bit difficult to follow. What if we remove the ariaLiveRegion property from the chart-plot? It is only used to render the LiveRegion component which can well be done from outside. To not increase the PR size this can be scoped to pie chart only